analyzer/codechecker_analyzer/analyzers/clangsa/analyzer.py

```diff
@@ -277,6 +277,9 @@ def analyzer_mentioned_file_real_path(self, mentioned_path):
                       mentioned_path, mentioned_path)
             return mentioned_path
 
+        mentioned_path = os.path.join(self.get_ctu_dir(), mentioned_path)
```

Comment:
Could this be at line 270? That way, the newly added redundant assignment for `suffix_index` won't be needed.

Reply:
Maybe one thing is worth to mention here: we assume if the path is relative then it must be relative to `self.get_ctu_dir()`. Is this always true?

Reply:
@martong Do we need to convert this file to absolute in case of On-demand CTU mode too?

Reply:
> Do we need to convert this file to absolute in case of On-demand CTU mode too?
Seems like for the tu_collector it is better to always provide the abolute paths.